pkg/progress/tty.go

```diff
@@ -158,7 +158,7 @@ func (w *ttyWriter) print() {
 	}
 
 	skipChildEvents := false
-	if len(w.eventIDs) > goterm.Height()-1 {
+	if len(w.eventIDs) > goterm.Height()-2 {
```

Comment:
Can you use a const with an explicit name for the magic `-2` number?

Reply:
ahh sorry, I was too quick on the merge 😓. I'll change it to something like `terminalOffset`, it's just to account for statuslines/other edge cases.